src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -170,17 +171,11 @@ public void onHeightMapUpdate(IColumn IColumn, int localX, int localZ, int oldHe
      */
     boolean relightMultiBlock(BlockPos startPos, BlockPos endPos, EnumSkyBlock type, Consumer<BlockPos> notify) {
         // TODO: optimize if needed
-
         // TODO: Figure out why it crashes with value 17
-        final int LOAD_RADIUS = 31;
+        final int LOAD_RADIUS = 17;
```

Comment:
Actually with this method it shouldn't need to check for loaded chunks at all. So why is it here?

Reply:
We still need to keep some cubes in cache. In that case `LOAD_RADIUS` used to keep some cubes around so we will not overuse `edgeNeedSkyLightUpdate` flag.

Reply:
ah, right. I forgot about this.